include/c_common/get_check_data.h

```diff
@@ -182,32 +182,32 @@ int64_t pgr_SPI_getBigInt(
         Column_info_t info);
 
 /*!
-@brief Function returns the value of specified column in double type.
-
-@param[in] tuple   input row to be examined.
-@param[in] tupdesc input row description.
-@param[in] info    contain column information.
-
-@throw ERROR Unexpected Column type. Expected column type is ANY-NUMERICAL.
-@throw ERROR When value of column is NULL.
-
-@return Double type of column value is returned.
-
+ * @brief Function returns the value of specified column in double type.
+ * 
+ * @param[in] tuple   input row to be examined.
+ * @param[in] tupdesc input row description.
+ * @param[in] info    contain column information.
+ * 
+ * @throw ERROR Unexpected Column type. Expected column type is ANY-NUMERICAL.
+ * @throw ERROR When value of column is NULL.
+ * 
+ * @return Double type of column value is returned.
+ * 
  */
 
 double  pgr_SPI_getFloat8(
         HeapTuple *tuple,
         TupleDesc *tupdesc,
         Column_info_t info);
 /*!
-@brief Function returns the string representation of the value of specified column.
-
-@param[in]  tuple   input row to be examined.
-@param[in]  tupdesc input row description.
-@param[in]  info    contain column information.
-
-@return Pointer of string is returned.
-
+ * @brief Function returns the string representation of the value of specified column.
+ * 
```

Comment:
This line has a trailing space:
There should be no space at the end of the line